data/blog/astral-projection-release-recap-sep-2023.mdx

```diff
@@ -0,0 +1,37 @@
+
+We even made a video to demo this particular use case:
+
+This gives you the benefits of a "privacy VPN," but at no additional cost and with basically zero configuration. Set up the Apple TV, approve its request to be an exit node from your admin console, and then access the internet from your phone, laptop, or even another Apple TV like you're sitting in your living room—no matter where you are.
+
+For many developers, it's not too much of a headache to spin up an inexpensive VPS through a cloud provider and use that as an exit node. We've heard about that workflow from lots of developers who need to access geo-restricted content or who are wrestling with sketchy local networks. (For example, you can read [my colleague Claire's report](https://tailscale.dev/blog/on-the-node-while-on-the-road) about bypassing some SSH shenanigans on hotel Wi-Fi.) One downside of this approach is that your connection will come from a commercial data center, and may be blocked, throttled, or subject to bot checks.
+
+Exiting from a residential network is more flexible, easier, and a little cheaper to boot. We're excited to see this released!
+
+### ...or browse from anywhere in the world while you're sitting at home
+
+**Parker**: In a fun bit of symmetry, the other big feature release this last month was [our integration with Mullvad](https://tailscale.com/blog/mullvad-integration/), now in beta. Mullvad is a privacy VPN that takes privacy seriously—and operates a huge fleet of high-speed exit nodes all around the globe. Starting this month, you can add a Mullvad subscription to your Tailscale account and get access to those nodes, making it a simple operation to route your traffic through basically anywhere in the world.
+
+That brings benefits in two major ways. If you don't already have a machine running as an exit node in your tailnet, this is an easy way to get one. You can select a nearby Mullvad node, count on a fast connection, and all of your traffic will get tunneled through WireGuard™ encryption between you and that node. But you can also pick a faraway node, and see the internet as if you were browsing from there. That may be useful for checking on CDN distribution, for localization workflows, for debugging geo-targeting, you name it.
+
+This feature came about in part because we've heard from lots of people who love Tailscale and love Mullvad, but don't love the hassle of trying to run two VPNs at the same time. If you think you might be interested in our Mullvad integration, check out [more information in our docs](https://tailscale.com/kb/1258/mullvad-exit-nodes/).
```

Comment:
```suggestion
This feature came about in part because we've heard from lots of people who love Tailscale and love Mullvad, but don't love the hassle of trying to run two VPNs at the same time. If you think you might be interested in our Mullvad integration, check out [our docs](https://tailscale.com/kb/1258/mullvad-exit-nodes/).
```